android/res/values/strings.xml

```diff
@@ -3,7 +3,9 @@
     <string name="app_name" translatable="false">QField</string>
     <string name="lib_name" translatable="false">qfield</string>
     <string name="done_unpacking_msg">Done unpacking, start QGIS.</string>
-    <string name="unpacking_msg">Unpacking post install data. This might take some time.</string>
+    <string name="unpacking_title">QField is finishing its installation</string>
```

Comment:
"Finalizing install" has the same meaning, but is much less wordy!

Reply:
@nyalldawson it was a placeholder, I was thinking of: "QField is getting ready for work"
what do you think?